faker/providers/misc/__init__.py

```diff
@@ -70,6 +71,13 @@ def language_code(cls):
         return cls.random_element(cls.language_codes)
 
     @classmethod
+    def uuid(cls):
```

Comment:
can you rename the moethod to be `uuid4`, so it's clear what kind of uuid is returning?